requirements-test.txt

```diff
@@ -2,7 +2,7 @@
 yarl
 mock>=1.0.1
 requests-mock>=1.3.0
-vcrpy>=1.11.1
+vcrpy<=2.0.0
```

Comment:
<= means including `2.0.0`, which is perhaps not the best as it will include the major version, but without any patch updates.

When protecting against major breaking upgrades you should use _exclusive lower than_ `<` and the version that's breaking.

Also, it appears to work, so you can actually do `< 3.0.0`

https://pip.pypa.io/en/stable/reference/requirement-specifiers/